http/oidc/pom.xml

```diff
@@ -32,6 +32,10 @@
 
     <artifactId>wildfly-elytron-http-oidc</artifactId>
 
+    <properties>
+        <version.org.apache.httpcomponents.httpcore>4.4.15</version.org.apache.httpcomponents.httpcore>
```

Comment:
Since the version is being added in the main pom.xml, we can remove it from here.